package.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+[ -e package ] && rm -r package
+mkdir -p package/opt
+mkdir -p package/usr/share/applications
+mkdir -p package/usr/share/icons/hicolor/scalable/apps
+
+cp -r dist/alinka package/opt/alinka
```

Comment:
That works with whitespace?

Reply:
Yes. It seems to be more readable with whitespaces.